add-on/src/lib/ipfs-proxy/pre-acl.js

```diff
@@ -18,8 +16,8 @@ function createPreAcl (getState, accessControl, origin, permission, requestAcces
     let access = await accessControl.getAccess(origin, permission)
 
     if (!access) {
-      const { allow } = await requestAccess(origin, permission)
-      access = await accessControl.setAccess(origin, permission, allow)
+      const { allow, remember } = await requestAccess(origin, permission)
+      access = await accessControl.setAccess(origin, remember ? '*' : permission, allow)
```

Comment:
Took me a moment to get it.. `remember` does not seem to be a good description for what is happening here :)
Perhaps `wildcard`  would be a more intuitive name than `remember`?

Reply:
👍